include/tallies/legendre_fet.hpp

```diff
@@ -12,6 +13,7 @@ using StaticVector6 = boost::container::static_vector<size_t, 6>;
 
 #include <iostream>
 #include <memory>
+#include <span>
```

Comment:
```suggestion
```
I don't think we are using std::span in this file, so we can remove this.

Reply:
I am sorry about this. we are not using it here. I don't how I missed it.